vagrant_resources/provisioners/provisioner_docker.sh

```diff
@@ -0,0 +1,3 @@
+sudo mv /etc/salt/minion{,-dist}
+sudo cp /vagrant/salt_resources/minions/minion.docker /etc/salt/minion
+sudo cp /vagrant/salt_resources/grains/grains /etc/salt/grains
```

Comment:
This is otherwise fine, but I'm wondering now if it is possible and wouldn't be better to include this in the Vagrantfile, so all the quick provisioning steps are easily seen in one place. Similar to the comment above consolidating the inline commands. Another ticket perhaps.